components/UniversalIdeasPage/UniversalIdeas.js

```diff
@@ -93,21 +93,21 @@ const StyledIdeaContainer = styled.div`
   display: grid;
   grid-template-columns: 80% 20%;
   margin: auto auto 10px auto;
+  padding: 10px 5px 10px 5px;
```

Comment:
```suggestion
  padding: 10px 5px;
```
glaube wenn du nur 2 Werte angibst, dann sind die oben/unten und rechts/links